includes/class-wp-job-manager-email-notifications.php

```diff
@@ -933,10 +934,12 @@ private static function get_email_content( $email_notification_key, $args, $is_p
 	 * @return string
 	 */
 	private static function inject_styles( $content ) {
-		if ( class_exists( 'Emogrifier' ) ) {
+		$css_inliner_class = CssInliner::class;
```

Comment:
So `CssInliner::class` is equal to `CssInliner`. So this code will either fail or it doesn't makes sense to check for existence.

Reply:
Just checked this today and `class_exists( CssInliner::class )` correctly returns false and doesn't throw an error if the class is not found. Agree that the `$css_inliner_class` variable is not needed, might even break things?

Reply:
(Apparently this is a compile-time construct, it's just returns the class FQN without the class needing to exists)
<img width="185" alt="image" src="https://github.com/Automattic/WP-Job-Manager/assets/176949/955f5d22-68c2-4608-9e8b-0eb4d8fa34f3">

Reply:
Yep the variable assignment was the one that I was worried about. I am amazed actually that it doesn't fail!

Still it would be easier to read without the variable.

Reply:
Good call!  done in b243532497362908204fcc2633842c1b1f93d347